lib/createlocaltracks.js

```diff
@@ -55,6 +58,26 @@ let createLocalTrackCalls = 0;
  *     console.log(localTrack.name);
  *   });
  * });
+ *
+ * @example
+ * // If you want to display your camera preview, pre-acquire media using
+ * // createLocalTracks. You can then pass these LocalTracks to connect.
```

Comment:
On second thought, this is not necessary because the comments below this already say the same thing.